tailwind.config.js

```diff
@@ -1,8 +1,12 @@
 const colors = require("tailwindcss/colors");
 
 module.exports = {
```

Comment:
There's a `theme.extend.fontFamily.sans` setting whose purpose I can't determine, and seems to do nothing, given that there's already a `font-sans` class. 😅

Reply:
This is extended so that we can later customize the font using a theme.